lib/md059.mjs

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+import { addErrorContext, stripAndDowncaseText } from "../helpers/helpers.cjs";
+import { validLink } from "../helpers/micromark-helpers.cjs";
+import { getReferenceLinkImageData, filterByTypesCached } from "./cache.mjs";
+
+const defaultBannedNames = [
```

Comment:
This should probably be a Set?

Reply:
Opted to change  [Line 37](https://github.com/DavidAnson/markdownlint/pull/1459/files#diff-6324bd4331648f082986c51ad43c2fe1770b5f0461eb4347c7dfe25262e3107aR37) to a set!